torch/_export/verifier.py

```diff
@@ -222,6 +222,10 @@ def _allowed_op_types() -> tuple[type[Any], ...]:
                 torch.amp.autocast_mode._enter_autocast,
                 torch.amp.autocast_mode._exit_autocast,
                 torch.fx.experimental.symbolic_shapes.cast_symbool_to_symint_guardless,
+                torch._functorch.vmap._add_batch_dim_python_wrapper,
+                torch._functorch.vmap._remove_batch_dim_python_wrapper,
+                torch._functorch.vmap._vmap_increment_nesting_python_wrapper,
+                torch._functorch.vmap._vmap_decrement_nesting_python_wrapper,
```

Comment:
So what does it take to remove this allowlist?

Reply:
I think similar story as subclasses serialization. We will eventually just joint IR trace the parts that are not seriaoizable and put Autograd HOP.